npc/re/merchants/odin_enchanter.txt

```diff
@@ -0,0 +1,367 @@
+//===== rAthena Script ======================================= 
+//= Soutanes Enchanter
+//===== By: ================================================== 
+//= mundussan
+//===== Current Version: ===================================== 
+//= 1.0
+//===== Compatible With: ===================================== 
+//= rAthena Project
+//===== Description: ========================================= 
+//= [Unofficial] 
+//= Merchant for enchanting Soutanes and Clergy equipment.
+//===== Additional Comments: ================================= 
+//= 1.0 First release. [mundussan]
+//============================================================ 
+
+hu_in01,97,323,4	script	Susan Grey#enchanter	893,{
```

Comment:
Are your npcs from replay?

Reply:
If with replay you mean taken from someone else, no. I used some parts from actual rAthena scripts and the rest I made by myself.

Reply:
Oh wait, if you mean the conversation of the npc, then yes, its from replay.
I can't play/understand kRO, so I took the dialogue from NovaRO.

Reply:
> .

I see. I don't have a replay recording from KRO. As I said, I used NovaRO dialogue to make my npc, and then I did my logic into it.

Reply:
You can use Google Lens to translate from your camera's phone directly :)